CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [VM/Network] Show IP addresses in front of their VIFs [#4882](https://github.com/vatesfr/xen-orchestra/issues/4882) (PR [#5003](https://github.com/vatesfr/xen-orchestra/pull/5003))
+- [Home/VM] Don't use sorted table for VMs (not) backed up (PR [#5046](https://github.com/vatesfr/xen-orchestra/pull/5046)
```

Comment:
```suggestion
- [Home/VM] Keep the home view on filtering by VMs (not) backed up (PR [#5046](https://github.com/vatesfr/xen-orchestra/pull/5046)
```

Reply:
IMHO, neither versions will be understandable by users.